db/migrate/20240909042900_create_products.rb

```diff
@@ -1,11 +1,11 @@
-class CreateProducts < ActiveRecord::Migration[7.0]
+class CreateProducts < ActiveRecord::Migration[6.0]
```

Comment:
The migration version has been downgraded from 7.0 to 6.0. Ensure this change is intentional and compatible with the rest of the application, as it may affect features available in the migration.